llms-install.md

```diff
@@ -0,0 +1,53 @@
+# Azure MCP Server Installation Guide
+
+This guide is specifically designed for AI agents install and configure the Azure MCP Server.
```

Comment:
Please add "to help" to make the sentence grammatically correct and clearer.

example: "This guide is specifically designed to help AI agents install and configure the Azure MCP Server."